<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChemDraw</class>
 <widget class="QMainWindow" name="ChemDraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>834</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ChemDraw</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/logo.png</normaloff>icons/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGraphicsView" name="pfdenv">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>170</red>
            <green>170</green>
            <blue>127</blue>
           </color>
          </brush>
         </property>
         <property name="foregroundBrush">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
         <property name="sceneRect">
          <rectf>
           <x>0.000000000000000</x>
           <y>0.000000000000000</y>
           <width>5000.000000000000000</width>
           <height>5000.000000000000000</height>
          </rectf>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::RubberBandDrag</enum>
         </property>
         <property name="transformationAnchor">
          <enum>QGraphicsView::AnchorUnderMouse</enum>
         </property>
         <property name="resizeAnchor">
          <enum>QGraphicsView::AnchorUnderMouse</enum>
         </property>
         <property name="viewportUpdateMode">
          <enum>QGraphicsView::FullViewportUpdate</enum>
         </property>
         <property name="optimizationFlags">
          <set>QGraphicsView::DontClipPainter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDockWidget" name="Icon_window">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>377</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>524287</width>
        <height>200</height>
       </size>
      </property>
      <property name="allowedAreas">
       <set>Qt::BottomDockWidgetArea</set>
      </property>
      <property name="windowTitle">
       <string>Icon</string>
      </property>
      <widget class="QWidget" name="icon">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="icon_tab">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_pipe">
           <attribute name="title">
            <string>Pipe</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QTabWidget" name="pipe_tab">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_straight">
               <attribute name="title">
                <string>Straight</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_elbow">
               <attribute name="title">
                <string>Elbow</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="tab_valve">
               <attribute name="title">
                <string>Valve</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_heat">
           <attribute name="title">
            <string>Heat Exchanger</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="4" column="1">
             <widget class="QLabel" name="label_heater">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Heater</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_cooler">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/Cooler.png</normaloff>icons/Cooler.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="shortcut">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_cooler">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Cooler</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_heater">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/Heater.png</normaloff>icons/Heater.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_pressure">
           <attribute name="title">
            <string>Pressure Chanager</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_reactor">
           <attribute name="title">
            <string>Reactor</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_columns">
           <attribute name="title">
            <string>Columns</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_solid">
           <attribute name="title">
            <string>Solid</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_user">
           <attribute name="title">
            <string>User Define</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDockWidget" name="log_window">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>524287</width>
        <height>150</height>
       </size>
      </property>
      <property name="allowedAreas">
       <set>Qt::BottomDockWidgetArea</set>
      </property>
      <property name="windowTitle">
       <string>Log</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="log_pan">
          <property name="font">
           <font>
            <family>Terminal</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Welcome to ChemDraw, an OpenSource P&amp;ID devlopment tool</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="status_lable">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="status_label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="status_label_3">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#bc13ff;&quot;&gt;ChemDraw  &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImport_as_PDf"/>
     <addaction name="actionImport_as_jpg"/>
     <addaction name="actionImport_as_png"/>
     <addaction name="actionImport_as_scg"/>
    </widget>
    <addaction name="actionSave_2"/>
    <addaction name="actionSave_As"/>
    <addaction name="menuImport"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionHelp_us"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionImport_as_PDf">
   <property name="text">
    <string>Import as PDf</string>
   </property>
  </action>
  <action name="actionImport_as_jpg">
   <property name="text">
    <string>Import as jpg</string>
   </property>
  </action>
  <action name="actionImport_as_png">
   <property name="text">
    <string>Import as png</string>
   </property>
  </action>
  <action name="actionImport_as_scg">
   <property name="text">
    <string>Import as svg</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionHelp_us">
   <property name="text">
    <string>Help us!</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
